frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -129,11 +137,27 @@ class AlertsContent extends Component {
     });
   };
 
+  handleAlertsLimitChange = (event) => {
+    this.setState({ alertsLimit: event.target.value });
+  }
+
   render() {
-    const { tab, allAlerts, unchecked, checked } = this.state;
+    const { tab, allAlerts, unchecked, checked, alertsLimit } = this.state;
     const { classes } = this.props;
     return (
       <div className={classes.root}>
+        <form align="center">
+          <InputLabel>View # of</InputLabel>
```

Comment:
also i assume this is the earlier "View # of alerts" textbox right? not the one shown in the demo :)

Reply:
Yup! Will make another PR for UI changes